projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/remote_data_job.py

```diff
@@ -0,0 +1,214 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+import time
+import uuid
+from enum import Enum
+from typing import Optional
+
+from taurus_datajob_api import ApiClient
+from taurus_datajob_api import Configuration
+from taurus_datajob_api import DataJobExecution
+from taurus_datajob_api import DataJobExecutionRequest
+from taurus_datajob_api import DataJobsExecutionApi
+from urllib3 import Retry
+from vdk.plugin.control_api_auth.authentication import Authentication
+
+log = logging.getLogger(__name__)
+
+
+class JobStatus(str, Enum):
+    """
+    Enum for the possible statuses a job execution can have.
+    """
+
+    SUBMITTED = "submitted"
+    RUNNING = "running"
+    SUCCEEDED = "succeeded"
+    CANCELLED = "cancelled"
+    SKIPPED = "skipped"
+    USER_ERROR = "user_error"
+    PLATFORM_ERROR = "platform_error"
+
+
+class RemoteDataJob:
+    """
+    Interact with Verstile Data Kit (VDK) Control Service
```

Comment:
that class should either implement/extend a core concept, or become a part of it itself...
this plugin seems to describe the domain model for the first time, that should be available to reuse in general - and to keep consistent; 
maybe have this as an additional effort

Reply:
Yes, I agree. Created https://github.com/vmware/versatile-data-kit/issues/1280 for this